apps/testing/src/main/java/com/akto/rules/CreateAdminUserViaMassAssignment.java

```diff
@@ -0,0 +1,180 @@
+package com.akto.rules;
+
+import com.akto.dto.ApiInfo;
+import com.akto.dto.OriginalHttpRequest;
+import com.akto.dto.OriginalHttpResponse;
+import com.akto.dto.RawApi;
+import com.akto.dto.testing.TestResult;
+import com.akto.dto.type.RequestTemplate;
+import com.akto.log.LoggerMaker;
+import com.akto.store.SampleMessageStore;
+import com.akto.store.TestingUtil;
+import com.akto.util.JSONUtils;
+import com.akto.util.enums.GlobalEnums;
+import com.mongodb.BasicDBObject;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+
+public class CreateAdminUserViaMassAssignment extends TestPlugin{
+
+    private final String testRunId;
+    private final String testRunResultSummaryId;
+    Logger logger = LoggerFactory.getLogger(CreateAdminUserViaMassAssignment.class);
+    private static final Set<String> allowedMethods = new HashSet<>(Arrays.asList("POST", "PUT", "PATCH"));
+    private static final List<String> allowedEndpointNames = Arrays.asList("users", "Users", "user", "User");
```

Comment:
can be done via regex as well instead of naming all combinations

Reply:
Lets do this via keywords for now, the current implementation doesn't cover all the cases, lets learn more from customers and later we will make this test even more comprehensive.